### PR TITLE
adding validation

### PR DESCRIPTION
**Pull request by iruzevic**
- added validation fixes for global variables
- added msg for missing global configuration
- adding steps how to set up global settings

## PLEASE DON'T REVIEW HUBSPOT PART 
this is still WIP

Screenshot 2021-10-06 at 09 36 16

Screenshot 2021-10-06 at 09 36 09

### CODE REVIEW

**Review by goranalkovic-infinum — APPROVED** [review]

**Review by iobrado — APPROVED** [review]

File: src/Rest/Routes/FormSubmitRoute.php [hunk 1]
```diff
@@ -157,6 +157,7 @@ public function routeCallback(\WP_REST_Request $request)
 			// Upload files to temp folder.
 			$files = $this->prepareFiles($files);
 
+			// Determin form type to use.
```
Comment (volfkarlo) [inline src/Rest/Routes/FormSubmitRoute.php:160]:
```suggestion
			// Determine form type to use.
```

File: src/Rest/Routes/FormSubmitRoute.php [hunk 2]
```diff
@@ -194,7 +195,7 @@ private function sendEmail(string $formId, array $params = [], $files = [])
 	{
 		$isUsed = (bool) $this->getSettingsValue(SettingsMailer::SETTINGS_MAILER_USE_KEY, $formId);
 
-		// If mailer system is not used just post as a success.
+		// If Mailer system is not used just post as a success.
```
Comment (volfkarlo) [inline src/Rest/Routes/FormSubmitRoute.php:198]:
```suggestion
		// If Mailer system is not used just respond with success.
```

**Review by volfkarlo — APPROVED** [review]
Micro corrections and a suggestion.

This is just a notice.
Did you make a check for already subscribed users?
On the Academy project we had problems with previously subscribed users. The problem arose when a user is created and then the same one wants to subscribe once more, you get an error message saying that the user is already created, that's why you have to update the user with the new mailing list, tag, or something similar to it.

File: src/Blocks/components/highlighted-content/highlighted-content-admin.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
+.es-highlighted-content {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	min-height: 300px;
```
[the remark this answers is not in the document]
Comment (iruzevic) [reply src/Blocks/components/highlighted-content/highlighted-content-admin.scss:5]: thx for the tip but this is WP admin settings and there is no responsive typography there so no need for this

**Comment by iruzevic** [comment]
> Micro corrections and a suggestion.
> 
> This is just a notice. Did you make a check for already subscribed users? On the Academy project we had problems with previously subscribed users. The problem arose when a user is created and then the same one wants to subscribe once more, you get an error message saying that the user is already created, that's why you have to update the user with the new mailing list, tag, or something similar to it.

Please open an issue with this problem, and write in more detail what needs to happen when the user is all-ready signed to the mailing list. Tnx

**Pull request merged**